src/pages/dashboard/charts/_ChoroLegend.js

```diff
@@ -45,17 +45,14 @@ export default function ChoroLegend({
     </g>
   )
   return (
-    <>
-      <h5 style={{ marginBottom: 0 }}>per million residents</h5>
```

Comment:
Dig removing the text, though we might want to make this a prop since all legends are going to need it